dev/import-beats/kibana.go

```diff
@@ -61,6 +68,9 @@ func (km *kibanaMigrator) migrateDashboardFile(dashboardFile []byte, moduleName
 		return nil, errors.Wrapf(err, "creating POST request failed")
 	}
 	request.Header.Add("kbn-xsrf", "8.0.0")
+	if km.authEnabled {
```

Comment:
Do you need authEnabled or could just check here if a username is set? -1 param like this.

Reply:
Fixed.